### PR TITLE
Redux form and StatusBar

### PR DESCRIPTION
**Pull request by felire**
### Summary 

I implemented the statusBar and i also implemented redux-form on the login screen. 

### Gifs 

transition7

### CODE REVIEW

File: WBooks/src/app/components/MyStatusBar/index.js [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
-  return (
-    <StatusBar
-      barStyle="dark-content"
-      hidden={false}
-      backgroundColor={white}
-      translucent={false}
-      networkActivityIndicatorVisible
-    />
-  );
+class MyStatusBar extends Component {
```
Comment (santinucera) [inline WBooks/src/app/components/MyStatusBar/index.js]: This should be a function

File: WBooks/src/app/screens/Addnew/index.js [hunk 2]
```diff
@@ -1,12 +1,39 @@
...
-    </View>
-  );
+import React, { Component } from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+import { withNavigation, NavigationActions, StackActions, SwitchActions } from 'react-navigation';
+import { ROUTES } from '@constants/routes';
+
+import { asyncStorageOperations } from './asyncStorageOperations';
+
+// const resetNavigatorAndGoToLogin = StackActions.reset({
```
Comment (santinucera) [inline WBooks/src/app/screens/Addnew/index.js]: Remove this comment

File: WBooks/src/app/screens/Addnew/index.js [hunk 3]
```diff
@@ -1,12 +1,39 @@
...
+    asyncStorageOperations.removeClient();
+    asyncStorageOperations.removeUserId();
+    navigate(ROUTES.Auth);
+  };
+
+  render() {
+    return (
+      <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+        <TouchableOpacity onPress={this.logOut}>
+          <Text>Log Out</Text>
```
Comment (santinucera) [inline WBooks/src/app/screens/Addnew/index.js]: This should be a constant

File: WBooks/src/app/screens/Login/layout.js [hunk 4]
```diff
@@ -1,41 +1,41 @@
...
-        <TextInput
-          placeholder={PASSWORD_PLACE_HOLDER}
-          secureTextEntry
-          onChangeText={text => updatePassword(text)}
-          {...COMMON_PROPS}
-        />
-      </View>
-      <ButtonOrLoading logInSuccessful={logInSuccessful} isLoading={isLoading} />
+      <Field
+        name="user"
```
Comment (santinucera) [inline WBooks/src/app/screens/Login/layout.js]: Field ñames should be cosntants

File: WBooks/src/app/screens/Login/layout.js [hunk 5]
```diff
@@ -1,41 +1,41 @@
...
+        component={InputComponent}
+        validate={[validEmail, emptyEmail]}
+        label={USER_PLACE_HOLDER}
+        onChange={onChange('user')}
```
Comment (santinucera) [inline WBooks/src/app/screens/Login/layout.js]: Same as above

File: WBooks/src/app/screens/Login/layout.js [hunk 6]
```diff
@@ -1,41 +1,41 @@
...
+      />
+      <Field
+        name="password"
```
Comment (santinucera) [inline WBooks/src/app/screens/Login/layout.js]: Same

File: WBooks/src/app/screens/Login/layout.js [hunk 7]
```diff
@@ -1,41 +1,41 @@
...
+        component={InputComponent}
+        validate={[emptyPassword, moreThanFiveLength]}
+        secureTextEntry
+        label={PASSWORD_PLACE_HOLDER}
+        onChange={onChange('password')}
```
Comment (santinucera) [inline WBooks/src/app/screens/Login/layout.js]: Same

File: WBooks/src/redux/auth/actions.js [hunk 8]
```diff
@@ -41,5 +41,8 @@ export const actionCreators = {
     const client = await asyncStorageOperations.getClient();
     const userId = await asyncStorageOperations.getUserId();
     login.setHeader(accessToken, client, userId);
+  },
+  removeError: () => dispatch => {
+    dispatch(privateActionsCreators.loginFailure({}));
```
Comment (santinucera) [inline WBooks/src/redux/auth/actions.js]: You should dispatch failure actions here, you should create an action that cleans the error

Comment (felire) [reply WBooks/src/redux/auth/actions.js]: But i'm dispatching failure actions there. Or Do I need to create a ResetAction and dispatch it?

**Pull request merged**